src/components/packages/Package-writing.svelte

```diff
@@ -0,0 +1,37 @@
+<style>
+    .text{
+        display: flex;
+        flex-direction: column;
+        width: 60%;
+        margin: 0 auto;
+        padding-left: 5px;
+        padding-right: 5px;
+    }
+
+    .text-body{
+        display:inline-block;
+        text-align: left;
+    }
+
+    h1{
+        font-family: 'Vollkorn SC', serif;
+        font-size: 8vh;
```

Comment:
The font size for "Packages" is larger than the "Pricing"  header in the PageBanner. Go ahead and copy the size and spacing from the Store page.
Also, the white space above and below "Packages" should be the same, check out the Store page for reference again.

<img width="1128" alt="Screen Shot 2020-10-22 at 4 30 38 PM" src="https://user-images.githubusercontent.com/42704409/96940337-53da8300-1484-11eb-94cd-2f198707c3af.png">

<img width="765" alt="Screen Shot 2020-10-22 at 4 31 26 PM" src="https://user-images.githubusercontent.com/42704409/96940385-753b6f00-1484-11eb-89b5-2bc508eaf634.png">